metaflow/datastore/s3_backend.py

```diff
@@ -220,26 +204,23 @@ def load_bytes(self, paths):
         if len(paths) == 0:
             return {}
```

Comment:
you should return a `BytesLoader`. The `with` statement fails otherwise

Reply:
Yep, already fixed :).